pkg/kine/drivers/sqlite/driver.go

```diff
@@ -95,7 +95,7 @@ var (
 			SELECT MAX(mkv.id) as id
 			FROM kine AS mkv
 			WHERE
-				mkv.name >= ? AND mkv.name < ?
+				mkv.name >= CAST(? AS TEXT) AND mkv.name < CAST(? AS TEXT)
```

Comment:
This cast is clearly necessary as we are passing a blob (and should be no-op for sqlite). However they are a bit arcane. Could we add a comment at the beginning of this `const (` section to explain why we are doing this?